web/src/narrow.js

```diff
@@ -181,7 +181,7 @@ export function activate(raw_operators, opts) {
         };
 
         const filter = new Filter(raw_operators);
```

Comment:
Rename `raw_operators`.

Also in this file: `adjusted_operators`, `adjusted_operators_if_moved(operators)`, and the `operators` variable in `to_compose_target`.